app.go

```diff
@@ -25,7 +25,7 @@ type app struct {
 	ui            *ui
 	nav           *nav
 	ticker        *time.Ticker
-	quitChan      chan bool
+	quitChan      chan struct{}
```

Comment:
What is the purpose of this change?

Reply:
I was originally running `preview` and `previewClear` in their own respective threads and synchronizing them through two `chan struct{}` variables. I changed all the instances of `chan bool` to be consistent with this. Using `chan struct{}` is idiomatic Go and it doesn't cause an allocation (since struct{} has a size of 0) whereas a `chan bool` will.